.github/workflows/check-data-package-version.yml

```
@@ -0,0 +1,37 @@
+name: Check Data Package standard version
+
+on:
+  schedule:
+    # Runs at midnight on Sundays
+    - cron: 0 0 * * 0
+  workflow_dispatch:
+
+env:
+  CURRENT_VERSION: v2.0
+  DP_REPO: frictionlessdata/datapackage
+
+jobs:
+  check-latest-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch latest version
+        run: |
+          latest_version=$(curl -s https://api.github.com/repos/$DP_REPO/releases/latest | jq -r .tag_name)
```

```suggestion
          latest_version=$(curl -s https://api.github.com/repos/$DP_REPO/releases/latest | jq -r .tag_name)
          latest_version=$(gh release list -R $DP_REPO --limit 1 --json name -q ".[].name"
```

Just a slightly clearer, slightly shorter version :stuck_out_tongue: mainly so I can use gh cli in more places :stuck_out_tongue: So this takes the top list of releases (latest) and gives back it's name.

---

Okay! I did it with `view` and `tagName`, but otherwise the same